R/metadata.R

```diff
@@ -51,5 +54,8 @@ xportr_metadata <- function(.df, metadata = NULL, domain = NULL) {
 
   ## End of common section
 
-  structure(.df, `_xportr.df_metadata_` = metadata)
+  structure(.df,
+    `_xportr.df_metadata_` = metadata,
+    `_xportr.df_verbose_` = verbose
```

Comment:
We could use quotes `"`, instead of backticks

```suggestion
    "_xportr.df_metadata_" = metadata,
    "_xportr.df_verbose_" = verbose
```